server/members/models.py

```diff
@@ -15,12 +16,15 @@ def validate_social_field(value):
     if not isinstance(value['isPrivate'], bool):
```

Comment:
Don't have to in this PR, but let's remove isPrivate; not worth the extra complexity